ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/LedgerClient.scala

```diff
@@ -79,6 +83,10 @@ final class LedgerClient private (
   val versionClient: VersionClient =
     new VersionClient(ledgerId, LedgerClient.stub(VersionServiceGrpc.stub(channel), config.token))
 
+  val userManagementClient: UserManagementClient = new UserManagementClient(
+    LedgerClient.stub(UserManagementServiceGrpc.stub(channel), config.token)
+  )
```

Comment:
IIRC this is already on `main` since yesterday, probably worth a rebase.

Reply:
Looks like someone was faster than me :rofl: